File: Input and Output & four arithmetic operations/C++/2588.cc
/* https://www.acmicpc.net/problem/2588 */
#include<iostream>
using namespace std;
int main() {
	//freopen("input.txt", "r", stdin);
	//freopen("output.txt", "w", stdout);
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);
	cout.tie(NULL);
	int a,b;
    int t;
    cin >> a >> b;
    t = b % 10;
    cout << a*t << '\n';
    t = (b % 100 - b % 10)/10;
    cout << a*t << '\n';
    t = (b % 1000 - b % 100)/100;
    cout << a*t << '\n';
    cout << a*b;
	return 0;
}

File: IO/Python/2588.py
#https://www.acmicpc.net/problem/2588
a = int(input())
b = int(input())
print(a*(b%10))
print(a*int(((b%100-b%10)/10)))
print(a*int(((b%1000-b%100)/100)))
print(a*b)